docs/.vuepress/config.ts

```diff
@@ -30,16 +33,10 @@ module.exports = {
       ],
     }],
     ['vuepress-plugin-code-copy', true],
-    [
-      'vuepress-plugin-typedoc',
-      {
+    ['vuepress-plugin-typedoc', {
         entryPoints: ['../../types/index.d.ts'],
         hideInPageTOC: true,
         tsconfig: 'tsconfig.json',
-        sidebar: {
-          fullNames: true,
-          parentCategory: 'API',
-        },
```

Comment:
Does this mean it does not show up in the sidebar anymore?

If so can't we put a @ts-ignore above it?